test/e2e/tests/nfts.spec.js

```diff
@@ -3,8 +3,8 @@ const { convertToHexValue, withFixtures } = require('../helpers');
 const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
 const FixtureBuilder = require('../fixture-builder');
 
-describe('Collectibles', function () {
-  const smartContract = SMART_CONTRACTS.COLLECTIBLES;
+describe('Nfts', function () {
```

Comment:
I think since this test is about NFTs and does not test a class or function, then this should be

``` suggestion
describe('NFTs', function () {
```